recipes/ckanweb-deploy-exts.rb

```diff
@@ -122,12 +122,21 @@
 
 	pluginname = "#{app['shortname']}".sub(/.*ckanext-/, "")
 
+	apprevision = app['app_source']['revision']
+	if ! apprevision
+		apprevision = "master"
+	end
+
 	# Don't install extensions not required by the batch node
 	#
 	installext = !batchnode || (batchnode && batchexts.include?(pluginname))
 	unless (!installext)
 		apprelease = app['app_source']['url'].sub 'http', "git+http"
 
+		if app['app_source']['type'].casecmp("git") == 0 then
```

Comment:
Not much point checking this when we've already updated `apprelease` on the previous line to be a Git URL.